news/2021-12-27-clickers-choice-2021.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: "Clicker's Choice: Top 20 osu! Players of 2021"
+date: 2021-12-27 09:00:00 +0000
+---
+
+Take part in helping the community determine the top 20 osu! players of 2021!
+
+![](/wiki/shared/news/2021-12-27-clickers-choice-2021/top_20_players_of_2021_banner.png)
+
+2021 was an incredible year for many circle clickers, some even with record-breaking performances in solo play and tournaments. As the year draws to a close, we're sending things off with the **9th iteration of the international top 20 voting**, at the community run [Clicker's Choice](https://choice.hwc.hr/) system!
+
+For those unaware, the osu! community has typically hosted a yearly top 20 vote on any number of platforms in the past with no real centralized place to handle anything. This year's top 20 vote will be run on the Clicker's Choice (community run) website, which is also home to the top player voting for Germany, Poland, Canada, UK and many other countries.
```

Comment:
I get that no official status should be implied, but there's really no need to state it twice.

```suggestion
For those unaware, the osu! community has typically hosted a yearly top 20 vote on any number of platforms in the past with no real centralized place to handle anything. This year's top 20 vote will be run on the Clicker's Choice website, which is also home to the top player voting for Germany, Poland, Canada, UK and many other countries.
```

Reply:
gotta wait for @peppy or @Ephemeralis on this one as they decide when to write community-run or not, but once this is solved everything should be ready